cfn-resources/cloud-backup-snapshot/test/inputs_1_create.template.json

```diff
@@ -1,10 +1,7 @@
 {
-    "GroupId": "",
+    "ProjectId": "",
     "ClusterName": "",
     "Description": "snapshotA through cloud formation template",
     "RetentionInDays": "5",
-    "ApiKeys": {
-        "PublicKey": "",
-        "PrivateKey": ""
-    }
+    "Profile":"default"
```

Comment:
The profile is not a required property. 
But if you don't pass anything `cfn test` would fail but it shouldn't. Right?